A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_AvailableArrivalsPage.js

```diff
@@ -54,7 +54,8 @@ class CDUAvailableArrivalsPage {
                             for (let j = 0; j < arrival.runwayTransitions.length; j++) {
                                 const runwayTransition = arrival.runwayTransitions[j];
                                 if (runwayTransition) {
-                                    if (runwayTransition.name.indexOf(selectedRunway) != -1) {
+                                    // Check if selectedRunway matches a transition on the approach (and also checks for Center runways)
+                                    if (runwayTransition.name.match("^RW" + selectedRunway + "(C)*$")) {
```

Comment:
```suggestion
                                    if (runwayTransition.name.match("^RW" + selectedRunway + "C?$")) {
```

Can be simplified to this (unless you want to match `RW09CCCCC`).

Reply:
Yeah this is much better. Tested it through a regex tester and looks like it works just as well :)